src/components/services/DeepLinkingManager.js

```diff
@@ -5,6 +5,7 @@ import * as React from 'react'
 import { Linking } from 'react-native'
 import { connect } from 'react-redux'
 
+import { YOLO_DEEP_LINK } from '../../../env.json'
```

Comment:
This needs to be `import ENV from '../../../env.json'`. According to the spec, JSON files only have a single default export, not named exports. Yes, the loosey-goosey RN packager accepts your syntax, but it technically shouldn't.